Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM node:16
+FROM node:19
 
+ENV TERM=xterm
+RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 COPY package*.json ./
-
 RUN npm install
 
-COPY . .
-
+COPY tsconfig.json ./
```

Comment:
Need to add another line `COPY .eslintrc ./` since we added linting to the prebuild.